webpack.config.js

```diff
@@ -1,30 +1,35 @@
-/**
```

Comment:
Much simpler!

🛑 It would be good to preserve some sort of comment explaining what the webpack.config.js file is doing now (definitely different from a typical config file).